src/index.html

```diff
@@ -1,13 +1,658 @@
+                  Контакти:
+                </h3>
+
+                <a
+                  href="https://goo.gl/maps/CAeSadQDCenzKFWM8"
+                  target="_blank"
+                  class="footer__addres"
+                >
+                  вул. M. Грушевського, 6; Київ, 01001 <br>
+                </a>
+
+                <a
+                  href="tel:+38 (044) 278-13-57"
+                  target="_blank"
+                  class="footer__addres"
+                >
+                 тел. +38 (044) 278-13-57<br>
+                </a>
+
+
+                <a
+                  href="tel:+38 (044) 278-74-54"
+                  target="_blank"
+                  class="footer__addres"
+                >
+                 факс +38 (044) 278-74-54<br>
+                </a>
+
+                <a
+                  href="mailto:info@namu.kiev.ua"
+                  target="_blank"
+                  class="footer__addres"
+                >
+                 e-mail. info@namu.kiev.ua<br>
+                </a>
+                <div class="footer__phone--up">
+                  <a href="#">
+                    <img
+                    src="./images/arrow.svg"
+                     alt="top"
+                     class="icon footer__up">
+                  </a>
+                </div>
+              </div>
+            </div>
+
+            <div class="
+              grid__item
+              grid__item--tablet-2-4
+              grid__item--desctop-8-10">
+              <ul class="footer__list">
+                <div class="footer__list--1">
+                  <li class="footer__item--top">
+                    <a
+                      href="#gallery"
+                      class="footer__link"
+                    > Галерея
+                    </a>
+                  </li>
+
+                  <li class="footer__item">
+                    <a
+                      href="#actions"
+                      class="footer__link"
+                    > Виставки та <br> події
+                    </a>
+                  </li>
+                </div>
+
+                <div class="footer__list--2">
+                  <li class="footer__item--top">
+                    <a
+                      href="#about-us"
+                      class="footer__link"
+                    > Про Hac
+                    </a>
+                  </li>
+
+                  <li class="footer__item">
+                    <a
+                      href="#contacts"
+                      class="footer__link"
+                    > Контакти
+                    </a>
+                  </li>
+                </div>
+              </ul>
+            </div>
+          </div>
+        </div>
+        <div class="footer__end">
+          <p class="footer__workers">
+            Designed by Zhyriada Marina
+          </p>
+          <p class="footer__workers">
+            Copyright © 2019. All Rights Reserved.
+          </p>
+        </div>
+      </div>
+    </footer>
```

Comment:
Move the contents of the <script> tag to an external JavaScript file and reference it in the HTML file using a script tag with src attribute.